### PR TITLE
Replace sprintf with format in templates

### LINKED ISSUE #1567: sprintf is used in some templates, format should be used
http://twig.sensiolabs.org/doc/filters/format.html

replace sprintf with format in the templates and depricate sprintf

### PR DESCRIPTION
**Pull request by carakas**
fixes #1567

### CODE REVIEW

File: src/Common/Core/Twig/Extensions/TwigFilters.php [hunk 1]
```diff
@@ -109,4 +109,20 @@ public static function getFilters(&$twig, $app)
...
+    /**
+     * Show deprecated error for sprintf.
+     *
+     * @return string
+     */
+    public static function deprecatedSprintf()
+    {
+        trigger_error(
+            '|sprintf is deprecated.
+             Use |format(args)|raw instead',
```
Comment (WouterSioen) [inline src/Common/Core/Twig/Extensions/TwigFilters.php]: can't sprintf be used without raw?

**Comment by WouterSioen** [comment]
👍

File: src/Common/Core/Twig/Extensions/TwigFilters.php (hunk 1 above)
Comment (carakas) [inline src/Common/Core/Twig/Extensions/TwigFilters.php]: our implementation of sprintf did the |raw automatically
Ill make it more clear that the |raw is only needed if you want to include html

Comment (carakas) [inline src/Common/Core/Twig/Extensions/TwigFilters.php]: added in ab83a50

**Comment by WouterSioen** [comment]
@carakas merge conflicts!

**Comment by carakas** [comment]
@WouterSioen fixed

**Pull request merged**